sgkit/stats/association.py

```diff
@@ -165,25 +129,36 @@ def gwas_linear_regression(
     Parameters
     ----------
     ds : Dataset
-        Dataset containing necessary dependent and independent variables
+        Dataset containing necessary dependent and independent variables.
     dosage : str
         Dosage variable name where "dosage" array can contain represent
         one of several possible quantities, e.g.:
         - Alternate allele counts
         - Recessive or dominant allele encodings
         - True dosages as computed from imputed or probabilistic variant calls
         - Any other custom encoding in a user-defined variable
-    covariates : Sequence[str]
+    covariates : Union[str, Sequence[str]]
```

Comment:
https://github.com/pystatgen/sgkit/issues/72?  I don't mind it since it's part of the VSCode template but I do end up with a somewhat awkward mix of conventions when I'm describing arrays vs anything else.  Although "ArrayLike" has been good IMO, especially when it comes with dimension descriptions the way numpy/dask do it.